vertx-mongo-client/src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java

```diff
@@ -18,12 +22,13 @@
 public class MongoClientOptionsParser {
 
   private final MongoClientSettings settings;
+  private final static CodecRegistry commonCodecRegistry = CodecRegistries.fromCodecs(new StringCodec(), new IntegerCodec());
```

Comment:
3 times BooleanCodec, is it for Boole birthday ? :-)
I will add those codecs and tests too.